# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tla/head4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/tla/tests/test-dir-regexps.sh	2004-01-22 04:08:02.000000000 +0100
+++ /tmp/tmp.vIxwK26605	2004-05-24 12:34:14.000000000 +0200
@@ -36,7 +36,7 @@
 echo "unrecognized ^attrapca$" > .arch-inventory
 test "$(echo $(tla inventory -u))" = "@rapca attrapca"
 echo "source ^@.*$" > .arch-inventory
-test "$(echo $(tla inventory -s --all | head -4))" = \
+test "$(echo $(tla inventory -s --all | head -n 4))" = \
      ".arch-ids/.arch-inventory.id .arch-inventory @rapca attrapca"
 end_test
 
